package.json

```diff
@@ -25,10 +25,12 @@
     "url": "https://github.com/integromat/imt-proto"
   },
   "license": "MIT",
+  "dependencies": {
+    "@types/request": "^2.48.12"
```

Comment:
The request type is part of the interface of the Hook. By not including the type in the dependencies, you get typescript error. This issue haven't surfaced yet, because most of the repositories depend on something (like express) that bring in this dependency. But by using the package in imt-blueprint it surfaced